src/components/ResourceFilters/README.md

```diff
@@ -0,0 +1,322 @@
+---
+name: Resource filters
+category: Lists and tables
+keywords:
+  - filters
+  - filtering
+  - filter control
+  - resource list
+  - index
+  - list filter
+---
+
+# Resource filters
+
+Resource filters is a composite component that filters the resource list component.
```

Comment:
We could just call this component `Filters`. It is already entirely possible to use it without a resource list.

Reply:
I was going to say - if this is just for use in ResourceLists should it live at `ResourceList.ResourceFilter`. If this can be used divorced from ResourceList can we see and example of it stand alone?

Reply:
Yes, if we keep this as-is we will alias it so it is also available at `ResourceList.Filters`. But it will stay as a top-level component for documentation reasons. We are going to do the same with `ResourceList.Item` and move it to a top-level component as `ResourceItem` so we can split up the `ResourceList` documentation.

Reply:
I think I'm leaning towards `Filters`, since this really could be used in any list context. Like you said, it is tied to Resource List context, but it doesn't need that in order to function. This component truly is flexible, so lets not use a naming convention that conceals that flexibility.

Reply:
Okay, I’ve made the change so this is now the `Filters` component and added an example of using it with data table.

Reply:
Nice! For the record - I dislike the ideas of creating aliases to top level components. I think we should avoid it to reduce potential for confusion. People will wonder "what's the difference between ResourceItem and ResourceList.Item" and the answer is there isn't one, they're identical.

Dragging ResourceList-specific subcomponents up into the top level because subcomponent documentation is a bit crap feels like the wrong solution to the problem - long-term I'd rather invest in making the styleguide render pages for public sub-components instead of dragging stuff up to the top level just so they get decent docs.

But that'll take some time so I'm happy to punt on that for the moment.